src/ngraph/runtime/reference/identity.hpp

```diff
@@ -0,0 +1,37 @@
+//*****************************************************************************
```

Comment:
We already have a kernel that does the same.
https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/reference/copy.hpp
I don't think we need a new kernel.

Reply:
removed